README.md

```diff
@@ -1,4 +1,4 @@
-# Dynomite-manager
+# Dynomite Manager
```

Comment:
I am curious about the name change... Why `Dynomite Manager` is better than `Dynomite-Manager`. The latter seems to provide a single name to the project. With your proposal, it is hard to understand if the project is called `Manager` or `Dynomite Manager`. I am not against it, but was wondering the reasoning ;)